packages/node_modules/@node-red/editor-client/locales/fr/jsonata.json

```
@@ -274,5 +274,9 @@
     "$clone": {
         "args": "valeur",
         "desc": "Cloner un objet en toute sécurité."
+    },
+    "$has": {
+        "args": "Set | Map, key",
+        "desc": "Vérifiez si une clé est présente dans un Set ou une Map. C'est une fonction définie par Node-RED."
```

```suggestion
        "desc": "Vérifie si la clé-valeur est présente dans l'objet `Set` ou `Map`. Ceci est une fonction définie par Node-RED."
```

I don't think the Node-RED ref is appropriate for `has`.

---

Why?
How would you solve the use cases I shared for "switch" and  "conditional assigment", while not using a function node?

---

I mean that description does not seem appropriate because it is a property of the object.

> This is a Node-RED defined function.

---

I copied that text from the $globalContext